assets/js/barChart.js

```diff
@@ -85,6 +104,23 @@ class BarChart {
                 .call(yAxis);
         }
 
+        // Add a tooltip to the y-axis labels.
+        let summaryData = formattedData;
```

Comment:
Typically defining a variable just to rename it isn't needed. If you think `summaryData` is a better name then just rename the original variable.